lib/src/lifecycle_module.dart

```diff
@@ -176,18 +184,75 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
       references.add(new Reference.followsFrom(_parentContext));
     }
 
-    return tracer.startSpan('LifecycleModule.$operationName',
-        references: references)
-      ..addTags({
-        'module.name': name,
-        'module.instanceId': _instanceId,
-      });
+    return tracer.startSpan(
+      'LifecycleModule.$operationName',
+      references: references,
+      tags: _defaultTags,
+    );
+  }
+
+  /// Creates a span with `globalTracer` from the start of [load] until now.
+  ///
+  /// This span is intended to represent the time it takes for the module to
+  /// finish asynchronously loading any necessary data and entering a state which
+  /// is ready for user interaction.
+  ///
+  /// Any [tags] or [references] specified will be added to this span.
+  @protected
+  void specifyEnterFirstUsefulStateTiming({
```

Comment:
Supercalifragilisticexpialidocious

Reply:
Do we want to have shorthands for each one, or should we just have consumers use `specifyStartupTiming(StartupTimingType.firstUseful)`?